apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -0,0 +1,175 @@
+@import "color", "font";
+
+/* This file is a adopted copy of shared/css/typography.scss
+  updating styles in this file will require a manual sync with the that file
+*/
+
+// TODO: [DSCO-64] (https://codedotorg.atlassian.net/browse
+//  /DSCO-64?atlOrigin=eyJpIjoiOTJkN2UxZDA0ZjViNGY4YWI5MGE0NDc2YWQyNWI0OGYiLCJwIjoiaiJ9)
```

Comment:
super minor: issue ID (DSCO-64) is probably fine, don't think we need the whole URL (or simplify to https://codedotorg.atlassian.net/browse/DSCO-64)